lang/strings.ts

```diff
@@ -0,0 +1,81 @@
+import { EOL } from 'https://deno.land/std@0.145.0/fs/eol.ts';
+import { isNil } from './values.ts';
+
+/**
+ * Removes a trailing new line (either LF or CRLF) from the string if there is one.
+ *
+ * @param text The text to be altered
+ * @returns The altered text
+ */
+export function trimTrailingNewline(text: string): string {
```

Comment:
So, this would remove just one trailing line, and no trailing spaces - The difference from `.trimEnd()` should be called out in the doc string I think.

Reply:
Done. Thanks!!